<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataFittingDialog</class>
 <widget class="QDialog" name="DataFittingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Fitting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSpectralDataFitting">
      <attribute name="title">
       <string>Spectral Data Fitting</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabThicknesses">
      <attribute name="title">
       <string>Thicknesses</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidgetThicknesses">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="tabRelativeErrors">
          <attribute name="title">
           <string>Relative Errors</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabAbsoluteErrors">
          <attribute name="title">
           <string>Absolute Errors</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabTable">
          <attribute name="title">
           <string>Table</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableView" name="thicknessTableView">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabRatesChart">
      <attribute name="title">
       <string>Rates Chart</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
